source/Walkthrough/virtual-networks.html.md.erb

```diff
@@ -297,6 +280,7 @@ Navigate to the [sbox.tfvar](https://github.com/hmcts/azure-platform-terraform/b
     backend_domain   = ["firewall-sbox-int-palo-labsgoldenpath<yourname>.uksouth.cloudapp.azure.com"]
     certificate_name = "wildcard-sandbox-platform-hmcts-net"
     disabled_rules   = {}
+    add_txt_record   = true
```

Comment:
I think we can get away without this, messaged you on slack, approving in the meantime though.